src/napari/_vispy/_tests/test_vispy_multiscale.py

```diff
@@ -196,8 +196,6 @@ def test_5D_multiscale(make_napari_viewer):
     """Test 5D multiscale data."""
     # Show must be true to trigger multiscale draw and corner estimation
     viewer = make_napari_viewer(show=True)
-    view = viewer.window._qt_viewer
-    view.set_welcome_visible(False)
```

Comment:
is it related to this? Why was this even here? This shouldn't be visible with layers present :/